### PR TITLE
lv2: fix memory leak and segfault

### PR DESCRIPTION
**Pull request by ndowens**
Patches from:
http://lv2plug.in/git/cgit.cgi/lv2.git/commit/?id=f8fdd7a48d12465d2ef8153d25337593c88687e3
http://lv2plug.in/git/cgit.cgi/lv2.git/commit/?id=dc9ff1250c049b5b60cef16f07be4c031ce2fa4c

### CODE REVIEW

File: srcpkgs/lv2/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-revision=1
-lib32disabled=yes
+revision=2
 build_style=waf
-homepage="http://lv2plug.in"
 hostmakedepends="python"
 makedepends="libsndfile-devel gtk+-devel"
 short_desc="Plugin standard for audio systems"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="LGPL-2.1, BSD"
```
Comment (maxice8) [inline srcpkgs/lv2/template]: please fix license

Comment (ndowens) [reply srcpkgs/lv2/template]: Didn’t change as I don’t recognize license in file https://github.com/drobilla/lv2/blob/master/COPYING  Didn’t want to change it without knowning the correct one to change it to

File: srcpkgs/lv2/template [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-lib32disabled=yes
+revision=2
 build_style=waf
-homepage="http://lv2plug.in"
 hostmakedepends="python"
 makedepends="libsndfile-devel gtk+-devel"
 short_desc="Plugin standard for audio systems"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-license="LGPL-2.1, BSD"
+license="LGPL-2.1, ISC"
```
Comment (maxice8) [inline srcpkgs/lv2/template]: LGPL-2.1 is -or-later or -only

Comment (ndowens) [reply srcpkgs/lv2/template]: Ah missed that one.

**Comment by ndowens** [comment]
Removed lgpl I did not find any mention in copyright nor in lv2.h file. What is in the lv2.h also lists isc

**Pull request merged**